bin/runtests.php

```diff
@@ -0,0 +1,490 @@
+<?php
```

Comment:
Nope -- this is the .php file. There is a separate file (runtests) with the she-bang in
it.

On Fri, Mar 25, 2016 at 05:15:15AM -0700, Wachter Johannes wrote:
>    In [1]bin/runtests.php:
> 
>  > @@ -0,0 +1,490 @@
>  > +<?php
> 
>    include the env line here: #!/usr/bin/env php
> 
>    —
>    You are receiving this because you authored the thread.
>    Reply to this email directly or [2]view it on GitHub
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu-io/sulu/pull/2184#discussion_r57439541
>    2. https://github.com/sulu-io/sulu/pull/2184/files/fe0af7bd20521c6e7cc60d5f84d163d3d6cf1997#r57439541
>    3. https://github.com/sulu-io/sulu/pull/2184/files/fe0af7bd20521c6e7cc60d5f84d163d3d6cf1997#r57439541